.palantir/revapi.yml

```diff
@@ -801,6 +801,11 @@ acceptedBreaks:
     - code: "java.method.removed"
       old: "method org.apache.iceberg.view.ViewBuilder org.apache.iceberg.view.ViewBuilder::withQueryColumnNames(java.util.List<java.lang.String>)"
       justification: "Acceptable break due to updating View APIs and the View Spec"
+    org.apache.iceberg:iceberg-core:
+    - code: "java.field.constantValueChanged"
```

Comment:
Another idea could be to add a new constant for new tables and persist the codec only during table creation, while still relying on the old `PARQUET_COMPRESSION_DEFAULT` for existing tables.

That way we won't have to change `SnapshotProducer` and encode old values.

Reply:
That being said, I am not against the current implementation.

Reply:
I had that earlier (not change the constant), I think @dbtsai wanted to do it the other way.